lib/formatters/table.js

```diff
@@ -0,0 +1,134 @@
+"use strict";
+
+var chalk,
+    table,
+    pluralize,
+    drawReport,
+    drawTable;
+
+chalk = require("chalk");
+table = require("table").default;
+pluralize = require("pluralize");
```

Comment:
I am assuming you are referring to the pluralize module? Yes, I have seen it inlined too (specifically, in the stylish formatter). However, inlining re-usable utility function logic to me sounds like unnecessary effort and additional surface to test. 

> On Jan 18, 2016, at 19:24, Gyandeep Singh <notifications@github.com> wrote:
> 
> In lib/formatters/table.js:
> 
> > @@ -0,0 +1,134 @@
> > +"use strict";
> > +
> > +var chalk,
> > +    table,
> > +    pluralize,
> > +    drawReport,
> > +    drawTable;
> > +
> > +chalk = require("chalk");
> > +table = require("table").default;
> > +pluralize = require("pluralize");
> Do we really need a dependency for this as I have seen in other area where we have just handled this by our own?
> 
> —
> Reply to this email directly or view it on GitHub.
>